File: ProjC1/src/projc1/Game/ScoreSystem.java
package projc1.Game;
class ScoreSystem {
    private int score;
    private int coinsCollected;
    
    public ScoreSystem() {
        this.score = 0;
        this.coinsCollected = 0;
    }
    
    public void addScore(int points) {
        this.score += points;
        this.coinsCollected++;
    }
    
    public void reset() {
        this.score = 0;
        this.coinsCollected = 0;
    }
    
    public int getScore() { return score; }
    public int getCoinsCollected() { return coinsCollected; }
}

File: ProcC1_pyver/game/components/score_system.py
class ScoreSystem:
    def __init__(self):
        self.score = 0
        self.coins_collected = 0
    
    def add_score(self, points):
        self.score += points
        self.coins_collected += 1
    
    def reset(self):
        self.score = 0
        self.coins_collected = 0
    
    def get_score(self): return self.score
    def get_coins_collected(self): return self.coins_collected